src/static/css/content.less

```diff
@@ -316,6 +322,19 @@ figure {
 // Initiate loop
 .color-swatch-loop( length(@colors) );
 
+.color-table {
+    width: 100%;
+    margin-bottom: 40px;
+
+    .swatch_field {
+        width: 200px;
+        height: 2.5em;
```

Comment:
I'm not sure I follow, or I'm not sure you follow. This is what I was envisioning:

![screen shot 2017-03-16 at 17 48 38](https://cloud.githubusercontent.com/assets/1044670/24020022/e45bc9a6-0a70-11e7-8f7a-4dbf23aabc73.png)

I just feels cleaner to me than the gray.